src/website/assets/scripts/chi-docs.js

```diff
@@ -174,6 +174,57 @@ onLoad(() => {
     enableCopyToClipboardFeature(codeSnippet);
   });
 
-  chi.dropdown(document.getElementById('version-dropdown'));
+  var dropdownButton = document.getElementById('version-dropdown');
+  var versions;
+  
+  dropdownButton.innerText = window.chiCurrentVersion;
+  chi.dropdown(dropdownButton);
   chi.dropdown(document.getElementById('support'));
+  
+  if(window.location.hostname === 'assets.ctl.io') {
+    var xmlhttp = new XMLHttpRequest();
+
+    xmlhttp.onreadystatechange = function() {
+        if (xmlhttp.readyState == XMLHttpRequest.DONE) {
+            if (xmlhttp.status == 200) {
+              versions = JSON.parse(xmlhttp.responseText);
+              fillDropdown();
+              checkChiCurrentVersion(versions[0]);
+            }
+        }
+    };
+
+
+    xmlhttp.open("GET", "https://assets.ctl.io/chi/versions.json", true);
+    xmlhttp.send();
+  }
+
+  function fillDropdown() {
+    var drop = document.getElementById('versionDropdown');
+
+    drop.innerHTML = "";
+    for (let version in versions) {
+      var versionAnchor = document.createElement('a');
+      
+      versionAnchor.setAttribute("href", "https://assets.ctl.io/chi/"+versions[version]);
+      versionAnchor.setAttribute("class", "m-dropdown__menu-item");
+      versionAnchor.innerText = "v" + versions[version];
+      drop.appendChild(versionAnchor);
+    }
+  }
+
+  // checkChiCurrentVersion("1.2.3");
+  function checkChiCurrentVersion(currentVersion) {
+      if(window.chiCurrentVersion.toString() !== currentVersion.toString()) {
+        var chiVersionCheckSelector = document.getElementById('chiVersionCheck');
+
+        chiVersionCheckSelector.innerHTML = `
+        <div class="m-alert -bubble -info" role="alert">
```

Comment:
Let's render this as a banner: `m-alert -banner -center -info` and place it above `o-header__content` with a 100% width `-w--100` so it stretches the entire width of the screen for maximum visibility. We'll need to apply a class on body or docs-container if the version isn't current and then use that class to adjust the padding-top of `docs-body` to `6.5rem` and `docs-body__aside` top value to `104px` to make room for the banner.

![image](https://user-images.githubusercontent.com/130654/66057699-07757b80-e4ee-11e9-8823-ddf39c697b32.png)